sdk/storage/azure-storage-file-datalake/tests/perfstress_tests/upload.py

```diff
@@ -0,0 +1,28 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from ._test_base import _FileTest
+
+from azure_devtools.perfstress_tests import RandomStream
+from azure_devtools.perfstress_tests import AsyncRandomStream
+
+
+class UploadTest(_FileTest):
```

Comment:
Same feedback as blob, download can use single service file, upload each instance must use own file.

Reply:
I think that's what it's already doing?